xrpl/models/transactions/vault_create.py

```
@@ -86,6 +86,14 @@ class VaultCreate(Transaction):
     domain_id: Optional[str] = None
     """The PermissionedDomain object ID associated with the shares of this Vault."""
 
+    scale: Optional[int] = None
+    """(Trust line tokens only) Specifies decimal precision for share calculations.
+    Assets are multiplied by 10^Scale to convert fractional amounts into whole number
+    shares. For example, with a Scale of 6, depositing 20.3 units creates 20,300,000
+    shares (20.3 × 10^Scale). For trust line tokens this can be configured at vault
```

Ruff (0.15.0)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)